plugins/main/ptp/ptp.go

```diff
@@ -245,9 +379,49 @@ func cmdAdd(args *skel.CmdArgs) error {
 		result.DNS = conf.DNS
 	}
 
+	if conf.SysCtl != nil {
```

Comment:
Ok interesting, so that part might be unnecessary, my need is to disable `rp_filter` on interfaces created within the container netns. If this is doable with the Mesos isolator, I could drop that part in my changes (would simplify a lot as we won't need to call the `tuning` plugin).

That being said, we must think of how is it configured for Kubernetes, so it might still be relevant to have this feature ?

Reply:
For Mesos, you we have some config in the policy if you can apply those configs for all kind of nodes (using CNI or not). If it should be applied only on nodes using the CNI plugin, you'll need to use our isolator path so that all remaining tunings for CNI nodes are at the same place.
For Kube, we only have tunings per clusters in Chef policy; we could put those tunings there if it should be applied on all nodes. We don't have per node logic for now but that could easily be done in chef too. We also don't know yet which CNI we will run to replace Flannel (maybe Cilium), so we can't say if we will use this part of the plugin or not, if the plugin would set those for us, etc....

As it's written already, I'm tempted to keep it, but it might also just never be used and it might be better to not spread this kind of config in several places too...

Maybe we can move this part in a separate PR to be merged later as we need it ? wdyt ?

Reply:
OK thanks for the explanation, I think these settings could be applied to all nodes (CNI or not).
I'll post that part in another PR then.